workers/unity/Assets/Playground/Scripts/Workers/UnityClient.cs

```diff
@@ -11,8 +11,17 @@ public class UnityClient : WorkerBase
 
         public override string GetWorkerType => WorkerType;
 
-        public UnityClient(string workerId, Vector3 origin) : base(workerId, origin)
+        private readonly ForwardingDispatcher forwardingDispatcher;
+
+        public UnityClient(string workerId, Vector3 origin) : base(workerId, origin, new ForwardingDispatcher())
+        {
+            forwardingDispatcher = (ForwardingDispatcher) View.LogDispatcher;
```

Comment:
How about `private readonly ForwardingDispatcher forwardingDispatcher = new ForwardingDispatcher();`
Then you can pass it along to `base` and drop the need for the cast and initialization here.

Reply:
When I attempt to do that I get: "An object reference is required for the non-static field, method, or property 'UnityClient.forwardingDispatcher'. Cannot access non-static field 'forwardingDispatcher' in static context."

Reply:
Ah well, I must have mis-remembered how that worked.